docs/getting-started/install-endpoint.asciidoc

```diff
@@ -1,70 +1,79 @@
 +
 2. On the Administration page of the {security-app} or the Endpoint Security integration page in Fleet, select **Add Endpoint Security**. The integration configuration page appears.
-3. Select a configuration for the Elastic Agent. You can use either the **Default config**, or add security integration to a custom or existing configuration. For more details on Elastic Agent configuration settings, see {fleet-guide}/elastic-agent-configuration.html[Configuration settings].
-4. Configure the Endpoint Security integration with a name and optional description. When the configuration is complete, select **Save integration**. Kibana redirects you back to the administration section of the {security-app}.
+3. Select an agent policy for the {agent}. You can select one of the two default policy options, an existing policy, or select **Create agent policy** to create a new one. For more details on Elastic Agent configuration settings, see {fleet-guide}/elastic-agent-configuration.html[Configuration settings].
+4. Configure the Endpoint Security integration with a name and optional description. When the configuration is complete, click **Save integration** in the lower right corner.
 +
 [role="screenshot"]
 image::images/install-endpoint/add-elastic-endpoint-security.png[]
-+
-5. On the "Enable Endpoint Security" on your Agent's page, select the name of your new integration. To enroll your agents with Endpoint Security, select **Enroll Agent**.
-6. Kibana redirects you back to Fleet to add the Elastic Agent to your host.
 
 [discrete]
 [[enroll-security-agent]]
 == Configure and enroll the Elastic Agent
 
-When integrating with the Elastic Agent, Endpoint Security **requires** enrollment through Fleet to enable the integration.
+To configure the {agent}, Endpoint Security requires enrollment through Fleet to enable the integration.
+
+IMPORTANT: Before you add an {agent}, a {fleet-server} must be running. See {fleet-guide}/fleet-server.html#add-fleet-server[Add a {fleet-server}].
+
+[discrete]
+[[fleet-server-upgrade]]
+=== Important information about the new {fleet-server}
+
+NOTE: If you are running an {stack} version earlier than 7.13.0, you can skip this section.
 
-IMPORTANT: Endpoint Security cannot be integrated with an Elastic Agent in Standalone mode.
+If you have upgraded to an {stack} version that includes the new {fleet-server} (>=7.13.0), you will need to redeploy your agents. Review the following scenarios to ensure you take the appropriate steps.
 
-1. Go to {fleet}. Select **Overview** > **Add agent**.
+* If you redeploy the {agent} to the same machine through the {fleet} application after you upgrade, a new agent will appear.
+* If you want to remove the {agent} entirely without transitioning to the {fleet-server}, then you will need to manually uninstall the agent on the machine. This will also uninstall the endpoint. See {fleet-guide}/uninstall-elastic-agent[Uninstall Elastic Agent].
```

Comment:
```suggestion
* If you want to remove the {agent} entirely without transitioning to the {fleet-server}, then you will need to manually uninstall the agent on the machine. This will also uninstall the endpoint. See {fleet-guide}/uninstall-elastic-agent.html[Uninstall Elastic Agent].
```